frontend/packages/dev-console/integration-tests/features/helm/actions-on-HelmRelease.feature

```diff
@@ -36,5 +55,4 @@ Scenario: Uninstall Helm Release through Context Menu: HR-08-TC04
     And user clicks on the "Uninstall Helm Release" action
     And user enters the release name
     And user clicks on the Uninstall button
-    # Then Helm release gets uninstalled - automatically navigating to Add page
-    Then user will be redirected to Topology page with no workloads
+    Then user will be redirected to Topology page with no workloads named "nodejs-ex-k"
```

Comment:
After creating the new namespace and adding helm release, If we uninstall the helm release, No workloads will be displayed. So I think you can use old statement itself